response_operations_ui/templates/edit-collection-exercise-details.html

```diff
@@ -0,0 +1,104 @@
+
+<html dir="ltr" lang="en-gb">
```

Comment:
We have a spoke template layout which could be used to reduce this file. 
You can look at `new-enrolment-code.html` to see how to use it